examples/builder/vsphere-iso/alpine/alpine-3.20.3.pkr.hcl

```diff
@@ -12,20 +21,21 @@ source "vsphere-iso" "autogenerated_1" {
   CPUs                 = 1
   RAM                  = 512
   RAM_reserve_all      = true
-  boot_command         = ["root<enter><wait>", "mount -t vfat /dev/fd0 /media/floppy<enter><wait>", "setup-alpine -f /media/floppy/answerfile<enter>", "<wait5>", "jetbrains<enter>", "jetbrains<enter>", "<wait5>", "y<enter>", "<wait10><wait10><wait10><wait10>", "reboot<enter>", "<wait10><wait10>", "root<enter>", "jetbrains<enter><wait>", "mount -t vfat /dev/fd0 /media/floppy<enter><wait>", "/media/floppy/SETUP.SH<enter>"]
+  boot_command         = ["root<enter><wait>", "mount -t vfat /dev/fd0 /media/floppy<enter><wait>", "setup-alpine -f /media/floppy/answerfile<enter>", "<wait5>", "VMw@re1!<enter>", "VMw@re1!<enter>", "<wait5>", "y<enter>", "<wait10><wait10><wait10><wait10><wait10><wait10>", "reboot<enter>", "<wait10><wait10>", "root<enter>", "VMw@re1!<enter><wait>", "mount -t vfat /dev/fd0 /media/floppy<enter><wait>", "/media/floppy/SETUP.SH<enter>"]
   boot_wait            = "15s"
   disk_controller_type = ["pvscsi"]
   floppy_files         = ["${path.root}/answerfile", "${path.root}/setup.sh"]
   guest_os_type        = "other3xLinux64Guest"
```

Comment:
The Linux Kernel for Alpine v3.20.3 is 5.2, correct? If so, this would be "other5xLinux64Guest" on vSphere 8.0 or later. Earlier, use "otherLinux64Guest".